pyproject.toml

```diff
@@ -1,47 +1,203 @@
 [build-system]
-requires = ["setuptools>=40.8.0", "wheel", "cython"]
-build-backend = "setuptools.build_meta:__legacy__"
+requires = ["setuptools>=64", "setuptools_scm>=8", "wheel"]
+build-backend = "setuptools.build_meta"
 
+[project]
+authors = [{name = "The HyperSpy Community"}]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Physics",
```

Comment:
Add the Physics topics also to extension libraries?
(included in https://github.com/hyperspy/rosettasciio/pull/196 )